server/src/main/java/org/candlepin/pinsetter/core/PinsetterKernel.java

```diff
@@ -67,6 +67,7 @@
 
     public static final String CRON_GROUP = "cron group";
     public static final String SINGLE_JOB_GROUP = "async group";
+    public static final String[] DELETED_JOBS = new String[] { "StatisticHistoryTask" };
```

Comment:
I think that allowing it to be configurable could get us out of a sticky situation where we missed adding to this array and released. Temp fix would be "Oh ya, we forgot that, just add it to the config file". No need to get a new build to fix the issue right away.

I mean, getting ourselves into that state wouldn't happen all that often, but, it would be nice to have something there just in case. Very simple to add.

I think that if we could as @alikins said, find a way to introspect what DELETED_JOBS should be, would be even better as we'd never have to worry about it again.

At the very least, I'd vote for a config option.